### ISSUE TITLE
Add `1.11.2.4-stretch` tag

### ISSUE THREAD

**Issue by amingilani** [openresty/docker-openresty#45]
Hi

I see that you've recently added stretch support, congratulations and thank you for your hard work!

However, most docker images based off of the openresty image should be tagged more specifically for compatibility reasons.

I see that [openresty doesn't use semver](https://github.com/openresty/openresty/issues/168) which already makes it difficult to understand which images are non-breaking. All the more reason to use a specific tag.

**Comment by neomantra** [comment]
There should have been an image named `1.11.2.4-stretch` built but it's not.  I thought I had checked that but apparently I failed.

I am triggering a build for it now and it should be on the Docker Hub in a few hours.  I will close this ticket then.

You are also pointing out something, that the specific DEB or RPM number is not tagged, which might be useful to some people.   The Docker Hub workflow won't do that automatically but I can make a script to do that.   But I probably won't be able to get to that for a while.
